src/ncp/ncp_base.hpp

```diff
@@ -588,12 +602,49 @@ class NcpBase
     uint8_t mPreferredRouteId;
 #endif
 
+    uint8_t mCurCommandIID;
+
 #if OPENTHREAD_RADIO || OPENTHREAD_CONFIG_LINK_RAW_ENABLE
+    uint8_t mCurTransmitIID;
     uint8_t mCurTransmitTID;
     int8_t  mCurScanChannel;
     bool    mSrcMatchEnabled;
 #endif // OPENTHREAD_RADIO || OPENTHREAD_CONFIG_LINK_RAW_ENABLE
 
+#if OPENTHREAD_CONFIG_MULTIPAN_RCP_ENABLE
+#if OPENTHREAD_RADIO || OPENTHREAD_CONFIG_LINK_RAW_ENABLE
+#define kPendingCommandQueueSize SPINEL_HEADER_IID_MAX
```

Comment:
Use constant variable instead of `#define`?
```suggestion
    static constexpr uint16_t kPendingCommandQueueSize = SPINEL_HEADER_IID_MAX`
```